src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -98,6 +98,11 @@ public void periodic() {
     SmartDashboard.putNumber("pitch", m_gyro.getPitch());
     SmartDashboard.putNumber("yaw", m_gyro.getYaw());
     SmartDashboard.putNumber("roll", m_gyro.getRoll());
+
+    SmartDashboard.putNumber("front left encoder", m_frontLeft.getPosition().angle.getDegrees());
+    SmartDashboard.putNumber("front right encoder", m_frontRight.getPosition().angle.getDegrees());
+    SmartDashboard.putNumber("rear left encoder", m_rearLeft.getPosition().angle.getDegrees());
+    SmartDashboard.putNumber("rear right encoder", m_rearRight.getPosition().angle.getDegrees());
```

Comment:
Having all these print statements is gonna slow everything down and is not really necessary outside of debugging purposes. If we need to debug something we can quickly write this code and then delete it once we figured out the problem. I don't see any reason for this to be on main.